labsc-notebooks/ERA5_series_moy_ann_sais_mens.ipynb

```diff
@@ -0,0 +1,3595 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ser_sais = pt.groupby(["time.season","time.year"]).mean() -273.15
</pre><p>using resample again <code>ser_sais = pt.resample(time="QS-DEC").mean(dim='time) = 273.15</code></p><p>Significance here is <strong>Q</strong>uarterly <strong>S</strong>tarting in <strong>DEC</strong>ember  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/LabSc/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>